cluster/addons/cluster-monitoring/prometheus/prometheus-deployment.yaml

```diff
@@ -0,0 +1,86 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
```

Comment:
Do you know what is prometheus cpu/memory usage and whether we can rely on defaults?